manifests/base/crds/flyteworkflow.flyte.net_flyteworkflows.yaml

```diff
@@ -0,0 +1,5152 @@
+
```

Comment:
This filename is auto-generated by `controller-gen`. should we rename it to `flyteworkflow.yaml`?
https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/templates/propeller/crds/flyteworkflow.yaml